tests/e2e/specs/login_spec.js

```diff
@@ -6,6 +12,21 @@ describe('Login', () => {
     password: 'p@ssword',
   };
 
+  const newCustomer = {
+    firstName: 'Andy',
+    lastName: 'Garcia',
+    email: 'f4831546-0062-470d-b6a9-cacb8e0a2aa4@mailslurp.com',
```

Comment:
Should this not be in .local.env. If may people run the test then this will get messy. Each dev can set up it's on mailslurp